### PR TITLE
advance-readings submission CESR-adapter-for-sophisticated-multisig Henk van Cann

### PR DESCRIPTION
**Pull request by henkvancann**
Hi Shanon,

The PR also contains the correction of a broken link in your root README.md.

Best, Henk

### CODE REVIEW

**Comment by henkvancann** [comment]
Added 'mailto:' to Will's e-mail address to overcome a broken link.

File: advance-readings/README.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ _Please add a level three header (`###`) for your paper's topic if it's not ther
    * One to two sentence synopsis or quote
    * #hashtags for topics
 ```
+
+
+### [Blockchainbird.org: CESR adapter for sophisticated multisig](https://hackmd.io/GbQO3p6QTge-8eQMGuMaeQ)
```
Comment (elenachachkarova) [inline advance-readings/README.md:51]: Could you use the paper name here, might be easier for people to find which paper this is (remove the 'Blockchainbird.org:' from the beginning). Thank you.

Comment (henkvancann) [reply advance-readings/README.md]: Done, feel free to correct minor things

**Comment by elenachachkarova** [comment]
[@henkvancann](https://github.com/henkvancann) Thanks for making the change and updating the broken link. 

@ChristopherA Everything looks correct.

**Comment by rhiaro** [comment]
Hi @henkvancann, I have attempted to send your discount code to the email address provided, but it bounced. Could you let me know the correct address to send it to? Thanks.

**Comment by henkvancann** [comment]
Dear Amy,

It should work on ***@***.*** You could cc; it to ***@***.***

Best, Henk

**Comment by rhiaro** [comment]
Hi @henkvancann - looks like github has censored that email address out. Could you please contact me at questions [at] weboftrust [.] info and I'll forward you the code that way?

**Pull request merged**